lib/config.js

```diff
@@ -47,6 +47,14 @@ const Config = {
       default: true,
       order: 3
     },
+    importNotebookResults: {
```

Comment:
Does this really need to be a setting ? If this works without bugs or some drawbacks, I don't this this can be default behaviour.

Reply:
I am sorry but I dont understand what that means. the reason for the setting is because it is rendered inline not the output pane and this could be undesired for many reasons so I made it optional. Also there are no draw backs and there should be very little impact since the only real change is adding a result bubble to the normal import procedure. I have no idea what bug could possibly prevent this from being a default implementation.

Reply:
Ah I'm sorry, horrible typos.
I tried to say "I think this can be default behavior" in the last sentence.

If you don't come up with any drawback, then let's make this the default and remove the config.


I feel like it would be better to keep configs clean as possible, since we have rather many.

Reply:
If an user don't like this, s/he can just
clear results.

And more, importing a notebook won't happen so much.

Reply:
Maybe you would understand better after local test because I could very easily see this as an annoyance. I know our config is a little messy, but I would rather clean that up rather than leave out a customization option.

Reply:
Okay sounds fair.
How about making a separate command for this feature ? i.e.: `Hydrogen: Import-Notebook-With-Results` will behaves like `importNotebookResults: true` while `Hydrogen: Import-Notebook` keeps behaving the same way as before.

I personally don't like changing configs so often and think using different command is easier.

Reply:
We can register those commands on `atom-workspace` and then we can call them from tree-view context menus of course ?

Reply:
Just add the new command to import with results here ...
https://github.com/nteract/hydrogen/blob/bd951e8a471dea932f5f5c7f0c27e8936f71a106/menus/hydrogen.cson#L3

Reply:
That won't allow you to set import with results to default though. But I agree it should still be added there.

Reply:
I firmly believe this should be a config because we cannot register two file openers when left clicking a file to open it, and therefore I am leaving this as implemented

Reply:
How about creating two commands for importing notebooks, and register them in package menus ?
Then we can right-click on tree-view and select either way: https://github.com/nteract/hydrogen/blob/master/menus/hydrogen.cson#L1

The default behaviour (which is invoked by left-click) can be configured via config tho.

Reply:
@aviatesk 
Roleplay:
So lets say I set default behavior with the config to be importing with results. Then the left click tree view uses that no problem and follows directions, but then what if I want to import via command palatte.
So I go and I run `hydrogen:import-notebook` and something weird happened, I see no results and I am confused, because I have my config to import results. I then realize that I have to call a completely separate command `hydrogen:import-notebook-with-results` and I think that is weird.
End Roleplay: I think to simplify for the user, we should be consistent with which version we are using. If someone wishes to load them without results, but does not want to switch it in the config. They will just have to clear results.

Reply:
Also I don't think that this is blocking, so even if we disagree now, after merging your could always open a new PR and show your use case. @aviatesk

Reply:
Fair points. Thanks for the detailed reply :)

> Also I don't think that this is blocking, so even if we disagree now, after merging your could always open a new PR and show your use case. @aviatesk

yep, definitely. But actually I don't have such a strong opinion on this, I just wanted to make a suggestion ?

So let's resolve this as is.